src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -198,7 +199,10 @@ void PerformAudioPassThruRequest::on_event(const event_engine::Event& event) {
     return;
   }
 
-  const ResponseParams response_params = PrepareResponseParameters();
+  ResponseParams response_params = PrepareResponseParameters();
+  if (message[strings::params].keyExists(hmi_response::message)) {
+    response_params.success = false;
```

Comment:
I am not sure this is the correct logic for handling if success is false. If the HMI sends a WARNINGS, SUCCESS, RETRY or the acceptable "success=true" related then success should be true. The HMI i believe can only send a message param with pertinent warning related messages if it is in the hmi error response structure.

Reply:
@OlhaVorobiova @dboltovskyi

Reply:
@VjKlepikov Could you support please

Reply:
@JackLivio, please find updates in [1f81d87](https://github.com/smartdevicelink/sdl_core/pull/3854/commits/1f81d87c448c4d5473a4fbd066355f70003f4242)